src/component/RenderJokes.jsx

```diff
@@ -0,0 +1,28 @@
+
+
```

Comment:
I think the empty spaces here are not necessary therefore they should be removed